examples/plotting/lon_lat_lines.py

```diff
@@ -16,38 +17,37 @@
 from sunpy.data.sample import AIA_171_IMAGE
 
 ###############################################################################
-# The purpose of this example is to demonstrate the coordinate transformations
-# that occur under the hood to show the heliographic grid lines of longitude
-# latitude. We first create the Map using the sample data.
+# We will demonstrate the coordinate transformations that occur under the
+# hood to show the heliographic grid lines of longitude latitude.
+# We first create a map using the sample data.
 
 aia = sunpy.map.Map(AIA_171_IMAGE)
 
 ###############################################################################
-# Let's first transform a single heliographic point coordinate.
+# Let's first specify a single heliographic point coordinate.
 
 stonyhurst_center = SkyCoord(12 * u.deg, 12 * u.deg,
                              frame=frames.HeliographicStonyhurst)
```

Comment:
```suggestion
```suggestion
lon_value = 35 * u.deg
lat_value = 12 * u.deg
stonyhurst_frame = frames.HeliographicStonyhurst(obstime=aia.date)
point_in_stonyhurst = SkyCoord(lon=lon_value, lat=lat_value, frame=stonyhurst_frame)
```